__tests__/generate.test.tsx

```diff
@@ -66,6 +66,78 @@ describe("GeneratePlaylistPage", () => {
     ).toBeInTheDocument()
   })
 
+  it('should display the error message when the "Create new playlist" option is selected and the playlist name is not filled', async () => {
+    render(<GeneratePlaylistPage {...staticTranslations} />)
+
+    const firstNavigationStepButton = screen.getByRole("button", {
```

Comment:
I see these first checks are repeated in all tests, which is redundant. This not only makes each tests check multiple things, which is inconvenient, but would also make all tests fail in case these first navigation steps fail.